sensors/barometer_console/barometer_console.ino

```diff
@@ -0,0 +1,111 @@
+#include <SPL06-007.h>
+#include <Wire.h>
+
+//#define Serial  SerialUSB
+
+void setup() {
+  Wire.begin();    // begin Wire(I2C)
+  Serial.begin(9600); // begin Serial
+
+  Serial.println("\nGoertek-SPL06-007 Demo\n");
+
+  SPL_init(); // Setup initial SPL chip registers - default i2c address 0x76  
+  // SPL_init(0x77); // Uncomment for alternate I2C address 0x77
+}
+
+void loop() {
+
```

Comment:
lots of prints like this could course issues if lots of data is collated over time. is this needed?